components/components.css

```diff
@@ -0,0 +1,591 @@
+  height: 35px;
+}
+
+.badge-top {
+  position: absolute;
+  border: 2px solid white;
+  top: 0px;
+  right: 3px;
+  width: 20px;
+  height: 20px;
+  background: red;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  border-radius: 50%;
+  font-size: small;
+}
+
+.badge-avatar {
+  position: relative;
+  display: flex;
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  justify-content: center;
+  align-items: center;
+  margin: 0 1rem;
+}
+
+.badge-bottom {
+  position: absolute;
+  border: 2px solid white;
+  bottom: -2px;
+  right: 1px;
+  width: 11px;
+  height: 11px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  border-radius: 50%;
+}
+
+.online {
+  background-color: rgb(4, 224, 4);
+}
+
+.offline {
+  background-color: rgb(224, 4, 4);
+}
+
+.btn {
+  border: none;
+  padding: 0.5rem 1rem;
+  cursor: pointer;
+  margin: 0.5rem;
+}
+
+.btn-sm {
+  width: 5rem;
+  word-wrap: break-word;
+}
+
+.btn-md {
+  width: 7rem;
+  word-wrap: break-word;
+}
+
+.btn-lg {
+  width: 9rem;
+  word-wrap: break-word;
+}
+
+.btn:hover {
+  /* border: 1px solid black; */
+  opacity: 0.9;
+}
+
+.btn:active {
+  transform: scale(0.98);
+}
+
+.btn-primary {
+  background-color: var(--primary-color);
+  border-radius: 0.25rem;
+}
+
+.btn-secondary {
+  background-color: var(--gray);
+  border-radius: 0.25rem;
+}
+
+.icon1 {
+  width: 15px;
+  height: 15px;
+}
+
+.btn-icon {
+  text-align: center;
+  background-color: var(--off-white);
+  border: 1px solid var(--dark-gray);
+  border-radius: 0.25rem;
+  padding: 0.5rem 1rem;
+}
+
+.float-action {
+  position: relative;
+}
+
+.btn-float {
+  position: fixed;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  bottom: 0.5rem;
+  right: 1rem;
+  font-size: x-large;
+  text-align: center;
+  border: none;
+  outline: none;
+  background-color: var(--dark-gray);
+  cursor: pointer;
+}
+
+.btn-float:hover {
+  transform: scale(1.1);
+}
+
+.btn-float:active {
+  background-color: var(--primary-color);
+}
+
+/* Cards */
+
+.card-list {
+  display: flex;
+  margin-bottom: 3rem;
+  flex-wrap: wrap;
+}
+
+.card {
+  border: 1px solid var(--gray);
+  position: relative;
+  width: 14rem;
+  display: flex;
+  flex-direction: column;
+  /* height: 375px; */
+  align-items: center;
+  justify-content: center;
+  margin: 0 1rem;
+}
+
+.card-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 100%;
+  background-color: white;
+}
+
+.card-img {
+  width: 100%;
+  /* height:280px; */
+}
+
+.card-title {
+  padding: 0;
+  margin: 0.3rem 0;
+}
+
+.card-description {
+  margin: 0.3rem 0;
+}
+
+.card-price {
+  margin: 0;
+  font-weight: bold;
+}
+
+.card-badge {
+  position: absolute;
+  top: 30px;
+  left: 0px;
+  background-color: var(--primary-color);
+}
+
+.card-dismiss {
+  position: absolute;
+  top: 0px;
+  right: 5px;
+  width: 20px;
+  height: 20px;
+  text-align: center;
+}

```

Comment:
add cursor as pointer to understand it is a clickable button.

Reply:
Added ```cursor: pointer``` to the dismiss button.